api/src/main/java/org/openmrs/module/kenyaemr/metadata/CommonMetadata.java

```diff
@@ -82,6 +82,14 @@ public static final class _EncounterType {
 		public static final String SARI_SURVEILLANCE = "76d55715-88cc-4851-b5e0-09136426fd46";
 		public static final String PROCEDURE_RESULTS = "99a7a6ba-59f4-484e-880d-01cbeaead62f";
 		public static final String NUTRITION = "160fcc03-4ff5-413f-b582-7e944a770bed";
+		public static final String AUDIOLOGY = "49da00fd-5b62-437a-a2d4-a28b3d22fa27";
+		public static final String PSYCHIATRIC = "7671cc06-b852-46e6-a279-afc8e2343a04";
+		public static final String ONCOLOGY = "70a0158e-98f3-400b-9c90-a13c84b72065";
+		public static final String PHYSIOTHERAPY = "a0ee267f-4555-48d7-9b1b-6d0dadee8506";
+		public static final String GOPC = "92be533c-35f0-4505-bfbd-95724bea0208";
```

Comment:
I realize this is post-merge, but for improved readability, it would be helpful to include the full name for GOPC and other abbreviations the first time they are mentioned.

Reply:
I agree @donaldkibet. Comments for each can also suffice